src/test/scala/cognite/spark/v1/SequencesRelationTest.scala

```diff
@@ -0,0 +1,133 @@
+package cognite.spark.v1
+
+import com.cognite.sdk.scala.common.CdpApiException
+import com.cognite.sdk.scala.v1.SequenceColumnCreate
+import io.scalaland.chimney.dsl._
+import org.apache.spark.sql.{DataFrame, Row}
+import org.apache.spark.sql.functions.col
+import org.apache.spark.SparkException
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.util.control.NonFatal
+
+class SequencesRelationTest extends FlatSpec with Matchers with SparkTest {
+  import spark.implicits._
+
+  private val sequencesSourceDf = spark.read
+    .format("cognite.spark.v1")
+    .option("apiKey", writeApiKey)
+    .option("type", "sequences")
+    .load()
+  sequencesSourceDf.createOrReplaceTempView("sequences")
+
+  it should "create and read sequence" in {
+    val key = shortRandomString()
+    val sequence = SequenceUpdateSchema(
+      externalId = Some("a"),
+      name = Some("a"),
+      columns = Some(Seq(
```

Comment:
How to do this with `spark.sql`?

Reply:
This is a good point. Turns out it's quite simple, I have added a test and example into the REAMDE.md - you just create `array(named_struct("externalId", "x", ...))`